packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -19,6 +19,47 @@ export enum VitalsUpdateFrequency {
     NEVER = 'NEVER'
 }
 
+export enum PropagatorType {
```

Comment:
Add description in JSDoc + warning to change this in the native code as well if we ever modify the value

Reply:
+ Move the PropagatorType into the resource tracking interface